pzsd_bot/cogs/points.py

```diff
@@ -32,142 +36,153 @@ async def on_message(self, message: Message) -> None:
         if message.author == self.bot.user:
             return
 
+        recipient_name = recipient_id = None
```

Comment:
why even declare recipient id here if it's declared on both side of the if statement

Reply:
haha yea you're right, I realized it after and figured you might catch it. I suppose i'll change it to just `recipient_name`.